<ui version="4.0" >
 <class>SymSolon</class>
 <widget class="QMainWindow" name="SymSolon" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SymSolon</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionModify" />
    <addaction name="actionSave" />
    <addaction name="actionClose" />
    <addaction name="separator" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionClassic_Scope" />
    <addaction name="actionSymbolon_Table" />
    <addaction name="separator" />
    <addaction name="actionTransits" />
    <addaction name="actionLife_Cyrcle" />
    <addaction name="actionSynastry" />
    <addaction name="separator" />
    <addaction name="actionBirth_date_refine" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>Windows</string>
    </property>
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_SymSolon" />
    <addaction name="actionAbout_Symbolon" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuSettings" />
   <addaction name="menuWindow" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConfigure" >
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionAbout_SymSolon" >
   <property name="text" >
    <string>About SymSolon</string>
   </property>
  </action>
  <action name="actionAbout_Symbolon" >
   <property name="text" >
    <string>About Symbolon</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="text" >
    <string>Paste</string>
   </property>
  </action>
  <action name="actionTile" >
   <property name="text" >
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade" >
   <property name="text" >
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionClassic_Circle" >
   <property name="text" >
    <string>Classic Scope</string>
   </property>
  </action>
  <action name="actionSymbolon_Table" >
   <property name="text" >
    <string>Symbolon Table</string>
   </property>
  </action>
  <action name="actionClassic_Rectangle" >
   <property name="text" >
    <string>Classic Rectangle</string>
   </property>
  </action>
  <action name="actionClassic_Boxes" >
   <property name="text" >
    <string>Classic Boxes</string>
   </property>
  </action>
  <action name="actionClassic_Scope" >
   <property name="text" >
    <string>Classic Scope</string>
   </property>
  </action>
  <action name="actionModify" >
   <property name="text" >
    <string>Modify</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Export</string>
   </property>
   <property name="visible" >
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTransits" >
   <property name="text" >
    <string>Transits</string>
   </property>
  </action>
  <action name="actionLife_Cyrcle" >
   <property name="text" >
    <string>Life Cyrcle</string>
   </property>
  </action>
  <action name="actionSynastry" >
   <property name="text" >
    <string>Synastry</string>
   </property>
  </action>
  <action name="actionBirth_date_refine" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Birth date refiner</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
